nx-X11/programs/Xserver/hw/nxagent/Display.c

```diff
@@ -2014,6 +2018,7 @@ Bool nxagentMakeIcon(Display *display, Pixmap *nxIcon, Pixmap *nxMask)
      }
   }
 
+  xfree(agentIconData);
```

Comment:
Why did you use xfree here, instead of the normal free or is the memcpy also from xserver?